_assets/stylesheets/application.scss

```diff
@@ -1,7 +1,25 @@
-@import "crds-styles-sprockets";
-@import "./fonts";
-@import "./utilities";
-// @import "./hotfix";
-@import "./components/modules";
-@import "./pages/modules";
-@import "./vendor/flickity";
+@import '~crds-styles/assets/stylesheets/variables';
```

Comment:
I'm unfamiliar with this style of import. What's happening here?

Reply:
First guess, this is a another Gem/package in the ☁️ ?

Reply:
@lukemoderwell The tilde represents a path to `node_modules`, using [this library](https://www.npmjs.com/package/node-sass-tilde-importer). That way we don't have to bring in vendor files but can use their npm package and still be able to use the sass `@import` idiom.